include/gsl/multi_span

```diff
@@ -1494,7 +1494,9 @@ public:
 
     constexpr reference operator[](const index_type& idx) const GSL_NOEXCEPT
     {
-        return data_[bounds_.linearize(idx)];
```

Comment:
From memory, `linearize()` already checks the condition that `idx` is within bounds. That is the point of the `bounds_` object. Why would we double-check the calculation here?